lib/runtime-core-tests/tests/imports.rs

```diff
@@ -142,6 +154,16 @@ fn imported_functions_forms(test: &dyn Fn(&Instance)) {
                 Ok(n + 1)
             }),
 
+            "callback_closure_polymorphic" => ErasedFunc::new_polymorphic(
```

Comment:
After offline chatting I think other name like `DynamicFunc::new` would make the API a bit clearer for the user.

This will imply:
* Renaming `ErasedFunc` to `DynamicFunc`
* Renaming the `new_polymorphic` method to just `new`

Thoughts @losfair ?

Reply:
```suggestion
            "callback_closure_polymorphic" => DynamicFunc::new(
```